src/runtime/thread_pool.cc

```diff
@@ -370,11 +374,7 @@ class ThreadPool {
   // number of workers used (can be restricted with affinity pref)
   int num_workers_used_;
   // if excluding worker 0 and using master to run task 0
```

Comment:
Let's also take the chance to revise this comment a bit, which even confused myself.

Suggested revision: //if or not to exclude worker 0 and use master to run task 0